deploy/02-deploy-time-lock.ts

```diff
@@ -10,16 +10,17 @@ const deployTimeLock: DeployFunction = async function (hre: HardhatRuntimeEnviro
   const { deployer } = await getNamedAccounts()
   log("----------------------------------------------------")
   log("Deploying TimeLock and waiting for confirmations...")
+  const args = [MIN_DELAY, [], []]
   const timeLock = await deploy("TimeLock", {
     from: deployer,
-    args: [MIN_DELAY, [], []],
+    args,
```

Comment:
```suggestion
    args: args,
```

Syntactical error.

Reply:
No this is not syntax error. This is ES^ shorthand> Please refer https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer

Reply:
That's fair. 

However, why not make it (`args:`) explicit to reduce ambiguity for debuggers (they will have to check what the parameter references to)?